src/lib/datepicker/calendar-body.spec.ts

```diff
@@ -117,22 +121,70 @@ describe('MatCalendarBody', () => {
 })
 class StandardCalendarBody {
   label = 'Jan 2017';
-  rows = [[1, 2, 3, 4, 5, 6, 7], [8, 9, 10, 11, 12, 13, 14]].map(r => r.map(createCell));
+  rows: MatCalendarCell<Date>[][];
   todayValue = 3;
   selectedValue = 4;
   labelMinRequiredCells = 3;
   numCols = 7;
 
+  constructor() {
+    const daysInMonth = daysInCurrentMonth();
+    const rows = this.createStackedNumbers(daysInMonth);
+
+    // We create rows based on the current month. This ensures that today is included.
+    this.rows = rows.map(r => r.map(this.createCell));
```

Comment:
I'm creating a month that covers as man days as this current month. I do this to ensure that the current calculation works. Today used to be passed in but now the calendar determines that itself. I just realized that I could simply do that by overriding the adapters `today()` method with a mock. 
I think I should revert this change.